pkg/controller/workload/workload_event.go

```diff
@@ -107,6 +108,30 @@ func (svc *ServiceEvent) processWorkloadResponse(rsp *service_discovery_v3.Delta
 	}
 }
 
+func deletePodFontend(uid uint32) error {
+	var (
+		err error
+		bk  = BackendKey{}
+		bv  = BackendValue{}
+		fk  = FrontendKey{}
```

Comment:
What the key and value stand for. Is it the serviceID? Or address?
I know it's the key and value of the map, but am confused as to what values it actually is.

Reply:
through the key{ip,port} to find service_id(service map key) or key{ip,0} to find backend_uid(bankend map key)